app/views/admin/sessions/index.haml

```diff
@@ -17,13 +17,16 @@
       %th Force Logout
     %tbody
       - @users.each do |user|
+        - last_login_activity = user.all_login_activities.successful.last
```

Comment:
Dont think this is very performant either, but this page was already doing a query like this to display `location_description`. If there are many active users, this could become a big n+1.